mendix/README.md

```diff
@@ -0,0 +1,39 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a market [leading low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
+
+With Mendix, you can easily create, change, integrate, test and deploy your applications. Moreover, you can manage branch lines and security as well as extend your app with custom code using the built-in editors.
+
+In addition to the native monitoring capabilities out of the box, Mendix also supports integrations to leading 3rd party application performance management vendors such as DataDog. Refer to our documentation to see all our [supported integrations][3].
+
+The Mendix integration with DataDog enables you to consume and monitor your Mendix ecosystem. The [DataDog integration][4] from Mendix will enable you to monitor
+Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics and any custom metrics you choose to
+configure in Mendix Studio Pro.
+
+
+> **Note**:  This integration is applicable to Mendix Cloud & Mendix Cloud Dedicated deployment models.
+
+
+## Setup
+
+To enable the DataDog integration for Mendix applications running on Mendix cloud follow the instructions [here][6].
+
+
+## Data Collected
+
+The list of metrics available for monitoring once the integration is enabled is detailed in the [Mendix documentation][7].
```

Comment:
If you're sending in metrics, you should include the metadata.csv file that lists the metrics being sent in.

For more info, refer [here](https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file).

Reply:
@pappachino Unfortunately, there isn't a way to export a list of metrics with their descriptions.

Reply:
If you include a list of metrics, we display it on our documentation site so customers can see a list of them on the same page and know what to expect. See an example under the 'Data Collected' section [here](https://docs.datadoghq.com/integrations/1e/).

Reply:
Thanks, @eho1307. I've dropped a metadata.csv file with all the generic metrics available from the Mendix integration.
We won't be able to list all of them, as the metric availability might vary from Mendix app to app.